microsoft-edge/progressive-web-apps-chromium/whats-new/pwa.md

```diff
@@ -254,7 +254,7 @@ To have more control over the title bar area that's currently displayed in stand
 
 Learn more about experimenting with Window Controls Overlay at [Experimental features in Progressive Web Apps (PWAs)](../how-to/window-controls-overlay.md).
 
-Register your origin for the **Web App Window Controls Overlay** trial at our [Origin Trials Developer Console](https://developer.microsoft.com/microsoft-edge/origin-trials/web-app-window-controls-overlay/registration/).
+Register your origin for the **Web App Window Controls Overlay** trial at our [Origin Trials Developer Console](https://developer.microsoft.com/microsoft-edge/origin-trials/web-app-window-controls-overlay/registration/).<!-- todo: update per https://microsoftedge.github.io/MSEdgeExplainers/origin-trials/ -->
```

Comment:
@captainbrosset @amandabaker - todo: update per https://microsoftedge.github.io/MSEdgeExplainers/origin-trials/

Reply:
I would remove this paragraph entirely. WCO isn't in origin trial anymore and has shipped long ago.

That being said, What's New articles tend to be snapshots in time. So, an alternative could be to add a pargraph in the intro that says something like "Sections of this document correspond to historical releases of Microsoft Edge and are not updated over time. References to new features or to origin trials might become outdated as new versions of Microsoft Edge get released."

What do you think is best? I see the value of the What's New pages being snapshots in time, so you could even re-visit them and see "oh this feature was in OT last year", or "oh this feature used to exist but is now deprecated". 

@amandabaker we could use your point of view here.

Reply:
I think DevTools What's New has a couple instances of such later notes.

Reply:
I agree that we should probably leave the old "What's New" pages as-is and add the note at the top, and then we should just make sure the individual feature docs are up to date. If we remove OT mentions from the feature docs and say that it's "Enabled by default in Edge [version]" then that should be sufficient to clear up any confusion.